next.config.js

```diff
@@ -1,3 +1,70 @@
-module.exports = {
-    useFileSystemPublicRoutes: false,
-}
\ No newline at end of file
+const path = require('path')
+
+const isProd = process.env.NODE_ENV === 'production'
+
+// paths
+const root = process.cwd()
+const src = path.resolve(root, './src')
+
+module.exports = {
+  useFileSystemPublicRoutes: false,
+  webpack(config, { isServer }) {
+    // Let webpack know about the theme alias to be able to use it in SCSS
+    // TS aliases should be set in the tsconfig
+    config.resolve.alias['theme'] = path.resolve(src, './theme')
+
+    // Add svgr support
+    config.module.rules.push({
+      // treat svgs in src/ as both react components and media files:
+      // https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs
+      test: /\.svg$/,
+      exclude: /node_modules/,
+      use: [
+        {
+          loader: '@svgr/webpack',
+          options: {
+            memo: true,
+          },
+        },
+        {
+          loader: 'file-loader',
+          options: {
+            name: '[name].[hash:8].[ext]',
```

Comment:
Should this be `contentHash` instead? IIRC, `hash` is the hash of the source file, `contentHash` is the hash of the final webpack output, which is what we care about. But this might have changed in the new webpack version

Reply:
I copied this from what we currently have, but I don't see any reason why we shouldn't use `contentHash` so will update this